templates/app/index.phtml

```diff
@@ -13,6 +13,7 @@ $this->headTitle('Welcome!');
 
         <h1>Our meetups...</h1>
         <p>Our meetups take place on every <strong>2<sup>nd</sup> Wednesday of the month from 7pm at the <a href="http://goo.gl/maps/ET78d">Oasis Conference Centre (Arundel Street)</a> in Portsmouth</strong>.</p>
+        <p>The next meetup will be happening on <?php echo $this->nextMeet ?> at 7pm</p>
```

Comment:
What if the meetup isn't happening (as happened last month)? I think this data should be sourced from the meetups data...